kornia/morphology/basic_operators.py

```diff
@@ -12,9 +12,9 @@ def _se_to_mask(se: torch.Tensor) -> torch.Tensor:
     num_feats = se_h * se_w
     out = torch.zeros(num_feats, 1, se_h, se_w, dtype=se.dtype, device=se.device)
     for i in range(num_feats):
-        y = i % se_h
-        x = i // se_h
-        out[i, 0, x, y] = (se_flat[i] >= 0).float()
+        y = i // se_w
+        x = i % se_w
+        out[i, 0, y, x] = (se_flat[i] >= 0).float()
```

Comment:
Do you have any snippet showing this issue ? We should add it to the unit tests to verify that we fix

Reply:
Could you quickly add few critical cases in the tests like this ?
https://github.com/kornia/kornia/blob/master/test/morphology/test_open.py#L19

Reply:
gotcha, added them to the erosion and dilation tests.

Reply:
does it affect to the others too ?

Reply:
I think every function in morphology is based on erosions and dilations?

Reply:
Yes, they are